crates/net/discv4/src/lib.rs

```diff
@@ -767,7 +768,7 @@ impl Discv4Service {
             // (e.g. connectivity problems over a long period of time, or issues during initial
             // bootstrapping) so we attempt to bootstrap again
             self.bootstrap();
-            return
+            return;
```

Comment:
please remove these semicolons, rustfmt should do this for you if you do `cargo +nightly fmt --all`. Note that this is nightly rustfmt, on stable I think it still adds semicolons